LayoutTests/accessibility/aria-combobox-no-owns.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../resources/js-test-pre.js"></script>
+<script src="../resources/accessibility-helper.js"></script>
+</head>
+<body id="body">
+
+<input type="text" role="combobox" id="combobox" aria-label="Combobox">
+<div role="list" id="list" aria-label="List1">
+  <div role="listitem" id="item1">item1</div>
+  <div role="listitem" id="item2">item2</div>
+</div>
+
+<p id="description"></p>
+<div id="console"></div>
+
+<script>
+    description("A combobox should still support aria-activedescendant even if it doesn't use aria-owns.");
+
+    if (window.accessibilityController) {
+        window.jsTestIsAsync = true;
+        var selectedChildrenChangeCount = 0;
+
+        window.accessibilityController.addNotificationListener(function (target, notification) {
```

Comment:
We might want to `accessibilityController.removeNotificationListener` right before `finishJSTest()` to clean this up. I'd hope this happens automatically between tests, but just to be safe might be good to include.